### PR TITLE
CSFML: Enable all x86 Linux platforms

### PR DESCRIPTION
**Pull request by Gnimuc**
pending #2950

### CODE REVIEW

**Comment by Gnimuc** [comment]
This needs to be re-triggered.

**Comment by giordano** [comment]
> This needs to be re-triggered.

You can do it :slightly_smiling_face:

**Comment by Gnimuc** [comment]
The only way I know is to do a dummy commit. I thought you have some magic spells to re-trigger the job. 🧙‍♂️

**Comment by giordano** [comment]
> I thought you have some magic spells to re-trigger the job.

I could have restarted only the failed jobs, but I think it's better to do a clean rebuild for all platforms.

**Comment by Gnimuc** [comment]
🧙‍♀️🧙🧙‍♂️

File: C/CSFML/build_tarballs.jl [hunk 1]
```diff
@@ -27,12 +27,15 @@ make install
...
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("i686", "linux"; libc="glibc"),
+    Platform("x86_64", "linux"; libc="musl"),
+    Platform("i686", "linux"; libc="musl"),
     Platform("x86_64", "macos"),
     Platform("i686", "windows"),
     Platform("x86_64", "windows"),
 ]
-platforms = expand_cxxstring_abis(platforms; skip=!Sys.iswindows)
+platforms = expand_cxxstring_abis(platforms)
```
Comment (Gnimuc) [inline C/CSFML/build_tarballs.jl:39]:
```suggestion


```

**Comment by giordano** [comment]
Note that this is still using obstack 1.1.0

File: C/CSFML/build_tarballs.jl [hunk 2]
```diff
@@ -27,12 +27,16 @@ make install
...
+
```
Comment (Gnimuc) [inline C/CSFML/build_tarballs.jl:39]:
```suggestion
```

**Pull request merged**